src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/SearchV3Controller.cs

```diff
@@ -62,6 +62,32 @@ public IHttpActionResult SearchApprenticeships(string keywords, int page = 1, in
             }
         }
 
+        /// <summary>
+        /// Search all apprenticeships
+        /// </summary>
+        /// <param name="searchString">String to search for</param>
+        /// <returns>a search result object</returns>
+        [SwaggerOperation("SearchActiveApprenticeshipsAutocomplete")]
```

Comment:
For a RESTful service you wouldn't normally have a verb  (Search) in the description.  That's the point of the http verb.  You would be getting a resource.  In this case a collection of Apprenticeships.  So would be /apprenticeship-programmes/Apprenticeships with a GET method that accepts a search string and returns a collection of apprenticeships based on that search string.

Reply:
because of the performance required for the autocomplete and the slimmed down payload required it was decided to implement a new api method as opposed to extending the other to take another query string param. As the original search function also suffers from this un-RESTful nature, it would be a bigger job to alter this method and regression test the multiple services which currently use it.